docs/source/reference/respawn/hud_element_notation.rst

```diff
@@ -0,0 +1,530 @@
+.. _element-notation:
+
+Element Notation
+====
+
+UI elements are created when a menu is initialized. References to the elements will stay the same, regardless if the menu is open or not.
+
+It is not possible to create elements at runtime so you have to define all elements a menu or panel contains beforehand in appropriate files.
+
+An Element is declared in the following way:
+
+.. code-block::
+
+    // please follow this structure
+    ElementName
+    {
+        ControlName name
+        // optional: classname, inheritance, ids ...
+
+        // optional: other properties
+
+        // optional: pinning
+    }
+
+If you're working on a **menu**, you need a ``menu`` object that contains all elements, for example like this:
+
+.. code-block::
+
+    resource/ui/menus/profiles_menu.menu
+    {
+        menu
+        {
+            ControlName Frame
+            xpos 0
+            ypos 0
+            zpos 3
+            wide f0
+            tall f0
+            autoResize 0
+            visible 1
+            enabled 1
+            pinCorner 0
+            PaintBackgroundType 0
+            infocus_bgcolor_override "0 0 0 0"
+            outoffocus_bgcolor_override "0 0 0 0"
+
+            // elements
+        }
+    }
+
+It usually doesn't matter if you use quotation marks to assign string values to parameters.
+
+.. cpp:function:: base
```

Comment:
What happened here?

Reply:
declaring a base is often used in vanilla HUD sheets but nobody knows what exactly that does and I forgot to look into it